internal/oci/oci_linux.go

```diff
@@ -1,10 +1,10 @@
+//go:build linux
```

Comment:
It would be nice to have a separate PR that adds go:build tags for all files at once, to minimize the noise introduced by it.

Also, in this particular case, the `linux` build tag is not needed as the file ends with `_linux.go`

So, something similar to https://github.com/opencontainers/runc/pull/3185 would be nice to have.

Reply:
hm this was added by vim-go plugin.. I agree we should probably do it all in one fell swoop, though I may push for this to stay if the plugin continues to add it :sweat_smile: